src/test/java/org/junitpioneer/jupiter/DefaultLocaleTests.java

```diff
@@ -63,24 +64,36 @@ void setsLocaleViaLanguageTag() {
 		}
 
 		@Test
-		@DefaultLocale(language = "en_EN")
+		@DefaultLocale(language = "en")
 		@DisplayName("sets the default locale using a language")
 		void setsLanguage() {
-			assertThat(Locale.getDefault()).isEqualTo(new Locale("en_EN"));
+			assertThat(Locale.getDefault()).isEqualTo(PioneerUtils.createLocale("en"));
 		}
 
 		@Test
 		@DefaultLocale(language = "en", country = "EN")
 		@DisplayName("sets the default locale using a language and a country")
 		void setsLanguageAndCountry() {
-			assertThat(Locale.getDefault()).isEqualTo(new Locale("en", "EN"));
+			assertThat(Locale.getDefault()).isEqualTo(PioneerUtils.createLocale("en", "EN"));
 		}
 
+		/**
+		 * A valid variant checked by {@link sun.util.locale.LanguageTag#isVariant} against BCP 47 (or more detailed RFC 5646) matches either {@code [0-9a-Z]{5-8}} or {@code [0-9][0-9a-Z]{3}}.
```

Comment:
Does it make sense to now add an additional test with the arguments we used before, and show that it fails non-compliant to BCP 47?